### ISSUE TITLE
Find alternative to kindlegen

### ISSUE THREAD

**Issue by ysb33r** [asciidoctor/asciidoctor-gradle-plugin#572]
With Amazoin discontinuing the old kindlegen, we need to find a new solution to the issue.

**Comment by abelsromero** [comment]
@slonopotamus  is working on something https://github.com/asciidoctor/asciidoctor-epub3/issues/363

**Comment by slonopotamus** [comment]
Yes, I am creating Ruby lib to write KF8 files. MOBI6 (legacy part of Mobi/KF8 joint file) is already there, but some bit are not there yet for proper KF8 support.
